packages/api/test/scripts/worker-globals.js

```diff
@@ -0,0 +1,6 @@
+export const caches = {
```

Comment:
Once we switch to sending ESM to cloudflare we can switch this to something we pass in explicitly to tests that need it, which'll is nice.